esmvaltool/config-references.yml

```diff
@@ -581,152 +581,11 @@ authors:
-  ghcn: "Jones and Moberg, J. Clim., doi:10.1175/1520-0442(2003)016<0206:HALSSA>2.0.CO;2, 2003."
-  hadcrut3: "Brohan et al., J. Geophys. Res., doi:10.1029/2005JD006548, 2006."
-  hadcrut4: "Morice et al., J. Goophys. Res., doi:10.1029/2011JD017187, 2012."
-  hadisst: "Rayner et al., J. Geophys. Res., doi:10.1029/2002JD002670, 2013."
-  hwsd: "Wieder et al., doi:10.3334/ornldaac/1247, 2014."
-  isccp-fh: "Zhang et al., Calculation, Evaluation and Application of Long-term, Global Radiative Flux Datasets at ISCCP: Past and Present, https://isccp.giss.nasa.gov/pub/flux-fh/docs/Zhang_etal_flux-cal_at-isccp_v3_2019.pdf, 2019."
-  jma-transcom: "Maki et al., Tellus B, doi:10.1111/j.1600-0889.2010.00488.x, 2010."
-  lai3g: "Zhu et al., Remote Sens., doi:10.3390/rs5020927, 2013."
-  landflux-eval: "Mueller et al., Hydrol. Earth Syst. Sci., doi:10.5194/hess-17-3707-2013, 2013."
-  landschuetzer2016: "Landschuetzer et al., Global Biogeochem. Cycles, doi:10.1002/2015GB005359, 2016."
-  mac-lwp: "Elsaesser et al., J. Clim., doi:10.1175/JCLI-D-16-0902.1, 2017."
-  merra2: "Gelaro et al., J. Clim., doi:10.1175/JCLI-D-16-0758.1, 2017"
-  mls-aura: "Read, W. and Livesey, N., MLS/Aura Level 2 Relative Humidity With Respect To Ice V004, Greenbelt, MD, USA, Goddard Earth Sciences Data and Information Services Center (GES DISC), doi:10.5067/Aura/MLS/DATA2019, 2015."
-  modis1: "Platnick et al., IEEE Trans. Geosci. Remote Sens., doi:10.1109/TGRS.2002.808301, 2003."
-  modis2: "Levy et al., Atmos. Meas. Tech., doi:10.5194/amt-6-2989-2013, 2013."
-  mte: "Jung et al., J. Geophys. Res., doi:10.1029/2010JG001566, 2011."
-  ncep: "Kalnay et al., B. Am. Meteorol. Soc., doi:10.1175/1520-0477(1996)077<0437:TNYRP>2.0.CO;2, 1996."
-  ndp: "Gibbs, H. K., doi:10.3334/CDIAC/lue.ndp017.2006, 2006."
-  niwa-bs: "Bodeker et al., Atmos. Chem. Phys., doi:10.5194/acp-5-2603-2005, 2005."
-  nsidc-0116: "Tschudi et al., Polar Pathfinder Daily 25 km EASE-Grid Sea Ice Motion Vectors, Version 4. [Indicate subset used]. Boulder, Colorado USA. NASA National Snow and Ice Data Center Distributed Active Archive Center. doi: https://doi.org/10.5067/INAWUWO7QH7B. [Date Accessed]."
-  osi-450: "EUMETSAT Global sea ice concentration climate data record 1979-2015 (v2.0). doi: 10.15770/EUM_SAF_OSI_0008, 2017."
-  patmos-x: "Heidinger et al., Bull. Amer. Meteor. Soc., doi:10.1175/BAMS-D-12-00246.1, 2013."
-  phc: "Steele et al., J. Clim., doi:10.1175/1520-0442(2001)014<2079:PAGOHW>2.0.CO;2, 2001."
-  piomas: "Zhang et al., Mon. Wea. Rev. doi:10.1175/1520-0493(2003)131<0845:MGSIWA>2.0.CO;2, 2013."
-  uwisc: "O'Dell et al., J. Clim., doi:10.1175/2007JCLI1958.1, 2008."
-  woa: "Locarnini et al., World Ocean Atlas 2013, Volume 1: Temperature, NOAA Atlas NESDIS 73, doi:10.7289/V55X26VD, 2013."
-
+# Note: the reference section has been replaced by the folder esmvaltool/references that contains bibtex files.
+# How to add a bibtex file to esmvaltool/references:
```

Comment:
It would be great if you could also add this explanation to the documentation on how to make a new recipe, so it becomes visible on readthedocs.